packages/storage/package.json

```diff
@@ -36,7 +40,10 @@
     "@firebase/app": "0.6.11",
     "@firebase/auth": "0.15.0",
     "rollup": "2.29.0",
+    "rollup-plugin-json": "4.0.0",
+    "rollup-plugin-sourcemaps": "0.6.3",
```

Comment:
You don't need sourcemaps and tmp, do you?

Reply:
No, removed.